packages/yourloops/test/integration/hcp-home-page.spec.tsx

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright (c) 2022, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import { Router } from 'react-router-dom'
+import { AuthContextProvider } from '../../lib/auth'
+import { MainLobby } from '../../app/main-lobby'
+import React from 'react'
+import { createMemoryHistory } from 'history'
+import { mockAuth0Hook } from './utils/mockAuth0Hook'
+import { mockNotificationAPI } from './utils/mockNotificationAPI'
+import { mockTeamAPI, teamThree, teamTwo } from './utils/mockTeamAPI'
+import { mockUserDataFetch } from './utils/auth'
+import { mockPatientAPI, monitoredPatient, removePatientMock, unMonitoredPatient } from './utils/mockPatientAPI'
+import { act, fireEvent, render, screen, within } from '@testing-library/react'
+import { checkHeader } from './utils/header'
+import { checkDrawer } from './utils/drawer'
+import { checkFooter } from './utils/footer'
+import { mockDataAPI } from './utils/mockDataAPI'
+import { mockDirectShareApi } from './utils/mockDirectShareAPI'
+import PatientAPI from '../../lib/patient/patient-api'
+import { checkPatientSecondaryBar } from './utils/patientSecondaryBar'
+
+describe('HCP can remove a patient from list', () => {
```

Comment:
This describe is way too specific, or the filename is not specific enough. It could simply be 'HCP home page' (and thereby improve the `it`s accordingly)